foil/Cargo.toml

```diff
@@ -0,0 +1,26 @@
+[package]
+name = "foil"
+version = "0.1.0"
+edition = "2021"
+authors = ["porcuquine <porcuquine@gmail.com>"]
+description = "Flat Optimization Intermediate Language"
+repository = "https://github.com/lurk-lab/lurk-rs"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+anyhow = { workspace = true }
+bellperson = { workspace = true }
+ff = { workspace = true }
+indexmap = { version = "1.9.3", features = ["rayon"] }
+generic-array = "0.14.7"
+lurk = { path = "../" }
+lurk-macros = { path = "../lurk-macros" }
```

Comment:
Not enough isolation

Reply:
I disagree, but will update to build and then make my case again.

Reply:
No need, it's fine. I only wrote that comment because we were transitioning from Alpha to LEM and I was concerned this PR would slow us down